.github/workflows/linux.yml

```diff
@@ -106,6 +110,7 @@ jobs:
           df -h
       - name: Run cmake
         run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
Because the checkout action, set one owner of the directory, and when inside the container, the user context changes. It seemed to me a better solution than changing ownership of the directory. See https://github.com/actions/runner/issues/2033